scripts/zones/Chateau_dOraguille/npcs/Halver.lua

```diff
@@ -102,7 +102,7 @@ function onTrigger(player,npc)
 		-- Mission San D'Oria 2-3 Journey Abroad 
 		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 11) then
 			player:startEvent(0x01fb);
-		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 0) then
+		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 1) then

```

Comment:
To fix var for players that started a quest or mission that got changed while they were in middle of it, use 
```
@setplayervar <player name> <variable name> <value>
```